overlord/devicestate/devicemgr.go

```diff
@@ -670,7 +670,11 @@ func (m *DeviceManager) Ensure() error {
 	var errs []error
 
 	if err := m.ensureSeeded(); err != nil {
-		errs = append(errs, err)
+		// XXX: avoid duplicated warnings?
```

Comment:
warnings have actually quite a bit of support already to avoid showing the same one again (default is not show the same for 24 hours), seems there's no access yet to control this but could be easily added. I suppose this will error again always the same the way?

Reply:
Yeah, it will always error the same way. In some sense it's probably good that this appears again and again because the underlying problem is not fixed and people who read the warning will have forgotten a couple of days ago. Of course it's bad to train people to ignore the warning by showing the same over and over again but then this should be super rare that this happens (or so we hope)

Reply:
do we need this XXX in the end?